decentraland/unity-renderer
# feat: minimap modifier for worlds

## What does this PR change?

Hides the minimap when going to a world. Add the Jump Back Home Button


## How to test the changes?

1. Enter Decentraland using the link https://play.decentraland.zone/?renderer-branch=feat/minimap-for-worlds&kernel-branch=feat/Jump-In-Home
2. Enter `/changerealm menduz.dcl.eth` in the chat
3. You are now in a World; a reef. The minimap should be gone, and you should see a Jump Home button in its place
4. Go back to `/changerealm loki`. The minimap should be back, and the Jump Home button should be gone
5. Enter Decentralnd using the link https://play.decentraland.zone/?renderer-branch=feat/minimap-for-worlds&kernel-branch=feat/Jump-In-Home&realm=menduz.dcl.eth. You should be directly in the World, minimap should be gone and button should be present


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
